### DISCUSSION TITLE
Can anyone confirm if there is text scaling in Wpf webview2

### DISCUSSION THREAD

**Discussion by timepasser00** [MicrosoftEdge/WebView2Feedback#4492] [Q&A] [answered]
I want to change the font size inside the webvew2. I don't want to zoom it.

**Comment by victorhuangwq** [comment]
@johna-ms is there a way to do that right now?

**Comment by johna-ms** [comment, marked as answer]
No there isn't. Zoom is the closest we have in the app, or there's also the option to use the OS setting to change font size for apps.

**Comment by timepasser00** [comment]
@johna-ms edge has the customize fonts feature. I also want something similar to that.
![image](https://github.com/MicrosoftEdge/WebView2Feedback/assets/59355713/5af2eb90-701c-47d0-8a62-e51d4c5ef896)

**Comment by johna-ms** [comment]
My mistake, Edge does offer a font resizing feature. But it remains that WebView2 does not. I'm editing the selected answer to be more accurate. Acknowledging here that originally I mistakenly said that Edge does not have a font resizing feature.

I believe a similar experience could also be achieved by dynamically setting the CSS style font-size.
